addons/web/controllers/main.py

```diff
@@ -1433,16 +1433,16 @@ def placeholder(image='placeholder.png'):
         '/web/content/<string:xmlid>',
         '/web/content/<string:xmlid>/<string:filename>',
         '/web/content/<int:id>',
-        '/web/content/<int:id>/<string:filename>',
-        '/web/content/<int:id>-<string:unique>',
-        '/web/content/<int:id>-<string:unique>/<string:filename>',
-        '/web/content/<int:id>-<string:unique>/<path:extra>/<string:filename>',
```

Comment:
any reason for dropping these alternative routes? In particular, the `/web/content/<int:id>/<string:filename>` looks like it may be useful outside of the context of assets, no?

Reply:
only the desire to simplify as much as possible. Do you want to add them back?

Reply:
@odony